services/pkgreview/package-rating.service.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { BaseJsonService } = require('..')
+
```

Comment:
We've got a lot of helper utility that encapsulate functionality common across many badges, and those can (and should) be used here as well to simplify the formatting.

```js
const { starRating, metric } = require('../text-formatters')
const { floorCount } = require('../color-formatters')
```

Reply:
Okay, so does this mean I would use the `starRating` text-formatter instead of the `starString` provided by the API?

Reply:
Correct, Shields will use the raw, numeric, values from the pkgreview.dev API response